editions/tw5.com/tiddlers/system/Sources.tid

```diff
@@ -1,26 +1,39 @@
 title: $:/editions/tw5.com/TiddlerInfo/Sources
 tags: $:/tags/TiddlerInfo
 caption: Sources
+code-body: yes
 
-\define static-link-base()
-https://tiddlywiki.com/static/$(title)$.html
+\function static-link-base() [[https://tiddlywiki.com/static/$(title)$.html]substitute[]]
+
+\function github-link-base()
+[[https://github.com/Jermolene/TiddlyWiki5/blob/tiddlywiki-com/editions/tw5.com/tiddlers/$(title)$]substitute[]]
 \end
 
-\define make-static-link()
+\procedure make-static-link()
+\whitespace trim
 <$set name="title" filter="[<currentTiddler>encodeuricomponent[]encodeuricomponent[]]" select="0">
-<a href=<<static-link-base>> class="tc-tiddlylink-external" target="_blank" rel="noopener noreferrer"><$text text=<<static-link-base>>/></a>
+	<a href=<<static-link-base>>
+		class="tc-tiddlylink-external"
+		target="_blank"
+		rel="noopener noreferrer"
+	>
+		<$text text=<<static-link-base>>/>
+	</a>
 </$set>
 \end
 
-\define github-link-base()
-https://github.com/Jermolene/TiddlyWiki5/blob/tiddlywiki-com/editions/tw5.com/tiddlers/$(title)$
-\end
-
-\define make-github-link()
-<$set name="title" value={{$:/config/OriginalTiddlerPaths##$(currentTiddler)$}}>
-<$set name="title" filter="[<title>encodeuricomponent[]]" select="0">
-<a href=<<github-link-base>> class="tc-tiddlylink-external" target="_blank" rel="noopener noreferrer"><$text text=<<github-link-base>>/></a>
-</$set>
+\procedure make-github-link()
+\whitespace trim
+<$set name="title" value={{{ [[$:/config/OriginalTiddlerPaths]getindex<currentTiddler>] }}}>
+	<$set name="title" filter="[<title>encodeuricomponent[]]" select="0">
+		<a href=<<github-link-base>>
+			class="tc-tiddlylink-external"
+			target="_blank"
+			rel="noopener noreferrer"
+		>
+			<$text text=<<currentTiddler>>/><span class="tc-tiny-gap-left">at github</span>
```

Comment:
I don't like the use of CSS to introduce spacing between two text items: it means that copying and pasting the text will not appear as expected. We should just be using `tc-tiny-gap-left` for spacing involving non-textual items such as icons.

Reply:
I would suggest that the link text be:

```
Direct link to "<$text text=<<currentTiddler>>/>" on github.com
```